sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/ScannerFileServiceTest.java

```diff
@@ -157,10 +154,10 @@ public void getFileByRelativePath_when_single_indexed_files_match_returns_file()
     Optional<InputFile> result = sut.getFileByRelativePath("/_/path/file.cs");
 
     assertThat(result).hasValue(expectedResult);
-    assertThat(logTester.logs(LoggerLevel.TRACE)).hasSize(2);
-    assertThat(logTester.logs(LoggerLevel.TRACE).get(1))
+    assertThat(logTester.logs(LoggerLevel.TRACE)).hasSize(1);
+    assertThat(logTester.logs(LoggerLevel.TRACE).get(0))
       .startsWith("Found indexed file ")
-      .endsWith("root/some/path/file.cs' for '/_/path/file.cs' (normalized to 'path/file.cs').");
+      .endsWith("sonar-dotnet/sonar-dotnet-shared-library/mod/root/some/path/file.cs' for '/_/path/file.cs' (normalized to 'path/file.cs').");
```

Comment:
this is relying on the behavior of `TestInputFileBuilder`, however I wanted to make sure the full path returned by the filesystem will be logged (and not some trimmed version)